src/argilla/client/feedback/integrations/huggingface/dataset.py

```diff
@@ -114,7 +114,7 @@ def _huggingface_format(dataset: Union["FeedbackDataset", "RemoteFeedbackDataset
 
         for record in dataset.records:
             for field in dataset.fields:
-                hf_dataset[field.name].append(record.fields[field.name])
+                hf_dataset[field.name].append(record.fields[field.name] if field.name in record.fields else None)
```

Comment:
Could we write like this?
```suggestion
                hf_dataset[field.name].append(record.fields.get(field.name, None))
```